pkg/docker/dockerfile/reverse/reverse.go

```diff
@@ -10,8 +10,9 @@ import (
 	"strings"
 	"time"
 
+	"github.com/docker/docker/api/types/container"
```

Comment:
Let's not pull in extra docker deps... The go-dockerclient package has the HealthConfig struct too: https://github.com/fsouza/go-dockerclient/blob/main/container.go#L381

Longer term (when we refactor this package) we'll have custom structs in the `pkg/docker/instruction` package

Reply:
Thanks for the review @kcq, I will remove it :)